api/client/service/update.go

```diff
@@ -39,6 +39,7 @@ func newUpdateCommand(dockerCli *client.DockerCli) *cobra.Command {
 	addServiceFlags(cmd, opts)
 
 	flags.Var(newListOptsVar(), flagEnvRemove, "Remove an environment variable")
+	flags.Var(newListOptsVar(), flagGroupRemove, "Remove additional user groups from the container")
```

Comment:
"Remove previously added user groups from the container"?

Not sure how to say this one... just a suggestion. Both sound weird.